Form/Handler/AuthorizeFormHandler.php

```diff
@@ -51,16 +51,6 @@ public function __construct(FormInterface $form, $requestStack = null)
         $this->requestStack = $requestStack;
     }
 
-    /**
-     * Sets the container.
-     *
-     * @param ContainerInterface|null $container A ContainerInterface instance or null
-     */
-    public function setContainer(ContainerInterface $container = null)
```

Comment:
Note: this is a BC break - if you want to have the changes in a patch release, you should instead deprecate the method.

Reply:
Agreed, we can't break BC here.